<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>CurveDetails</class>
    <widget class="QMainWindow" name="CurveDetails">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>500</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>400</width>
                <height>500</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>400</width>
                <height>500</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QLabel" name="nodes_label">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>50</y>
                        <width>231</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>PLACEHOLDER</string>
                </property>
            </widget>
            <widget class="QLabel" name="label">
                <property name="enabled">
                    <bool>true</bool>
                </property>
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>20</y>
                        <width>58</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Name:</string>
                </property>
            </widget>
            <widget class="QLabel" name="name_label">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>20</y>
                        <width>231</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>PLACEHOLDER</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_2">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>50</y>
                        <width>58</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Nodes:</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_3">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>85</y>
                        <width>58</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Color:</string>
                </property>
            </widget>
            <widget class="QComboBox" name="comboBox">
                <property name="geometry">
                    <rect>
                        <x>80</x>
                        <y>80</y>
                        <width>91</width>
                        <height>32</height>
                    </rect>
                </property>
                <item>
                    <property name="text">
                        <string>blue</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>red</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>green</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>black</string>
                    </property>
                </item>
            </widget>
            <widget class="QLabel" name="label_4">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>140</y>
                        <width>61</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Translate:</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_5">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>180</y>
                        <width>58</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Scale:</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_6">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>220</y>
                        <width>58</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Rotate:</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="translate_dx">
                <property name="geometry">
                    <rect>
                        <x>120</x>
                        <y>140</y>
                        <width>31</width>
                        <height>21</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLineEdit" name="translate_dy">
                <property name="geometry">
                    <rect>
                        <x>160</x>
                        <y>140</y>
                        <width>31</width>
                        <height>21</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLineEdit" name="scaleInput">
                <property name="geometry">
                    <rect>
                        <x>120</x>
                        <y>180</y>
                        <width>71</width>
                        <height>21</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLineEdit" name="rotateInput">
                <property name="geometry">
                    <rect>
                        <x>120</x>
                        <y>220</y>
                        <width>71</width>
                        <height>21</height>
                    </rect>
                </property>
            </widget>
            <widget class="QPushButton" name="translateApply">
                <property name="geometry">
                    <rect>
                        <x>230</x>
                        <y>130</y>
                        <width>112</width>
                        <height>32</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Apply</string>
                </property>
            </widget>
            <widget class="QPushButton" name="scaleApply">
                <property name="geometry">
                    <rect>
                        <x>230</x>
                        <y>170</y>
                        <width>112</width>
                        <height>32</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Apply</string>
                </property>
            </widget>
            <widget class="QPushButton" name="rotateApply">
                <property name="geometry">
                    <rect>
                        <x>230</x>
                        <y>210</y>
                        <width>112</width>
                        <height>32</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Apply</string>
                </property>
            </widget>
            <widget class="QListWidget" name="nodesList">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>260</y>
                        <width>341</width>
                        <height>231</height>
                    </rect>
                </property>
            </widget>
            <widget class="QCheckBox" name="hiddenInput">
                <property name="geometry">
                    <rect>
                        <x>280</x>
                        <y>20</y>
                        <width>86</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>hidden</string>
                </property>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
